.github/workflows/tests.yml

```diff
@@ -35,14 +33,17 @@ jobs:
                 command: |
                   composer remove --dev --ignore-platform-reqs phpunit/phpunit
                   composer require --dev --ignore-platform-reqs --update-with-all-dependencies phpunit/phpunit:^7
+            - if: ${{ matrix.php == '5.6' || matrix.php == '7.0' || matrix.php == '7.1' }}
+              name: Run PHPUnit Patches
+              run: sh .github/apply-phpunit-patches.sh
             - name: Run Script
               run: vendor/bin/phpunit
     test_lowest:
-        runs-on: ${{matrix.operating-system}}
+        runs-on: ubuntu-latest
```

Comment:
This is better, avoids repetition from `operating-system: [ ubuntu-latest ]` ?
`runs-on: ${{matrix.operating-system}}`

Reply:
As the matrix only applies to one test (and not to all of them), it doesn't save us any repetition, it only adds an extra line of code.